paddle/fluid/operators/recurrent_op.h

```diff
@@ -159,7 +165,9 @@ class RecurrentBase : public framework::OperatorBase {
     if (is_backward && src_var == nullptr) {
       return;
     }
-    PADDLE_ENFORCE_NOT_NULL(src_var, "%s is not found.", src_var_name);
+    PADDLE_ENFORCE_NOT_NULL(
+        src_var, platform::errors::InvalidArgument(
+                     "Source variable %s is not found.", src_var_name));
```

Comment:
May be better to use `platform::errors::NotFound` instead of `platform::errors::InvalidArgument`  and the same of other

Reply:
Done